docs/user/quickstart.md

```diff
@@ -16,26 +16,29 @@ pip install .
 
 You can also install it in the editable mode with `pip install -e .`.
 
-* __Dependencies:__ Hivemind requires python 3.7+ (3.8 is recommended), it will install [requirements](https://github.com/learning-at-home/hivemind/blob/master/requirements.txt) automatically; 
-* __OS support:__ Linux and macOS should [just work](https://github.com/learning-at-home/hivemind/issues).
-We do not officially support Windows, but you are welcome to contribute your windows build :)
+* __Dependencies:__ Hivemind requires Python 3.8+.
```

Comment:
@leshanbog mentioned that all but one tests are operational on py37
![image](https://user-images.githubusercontent.com/3491902/109421720-8cb5d600-79e9-11eb-8f8a-6c113d3c6e17.png)

So we have two options
- keep it at py3.7
- change, but revert after @leshanbog implements the fix

Reply:
I guess the second option is preferable, since right now some parts of the code do not work